caluma/caluma_form/validators.py

```diff
@@ -130,6 +130,14 @@ def _evaluate_options_jexl(
             # and use the structure code)
             return [o.slug for o in question.options.all()]
 
+        elif not validation_context:
+            # First see if we really need to build up the validation context. If
+            # not, we can speed things up significantly
+            all_options = list(question.options.all())
+            if all(opt.is_hidden in ("", None, "false") for opt in all_options):
```

Comment:
Can `.is_hidden` even be an empty string or `None`?

Reply:
You're right, it can't be `None`, and the default is `"false"`. I'd still take the empty string as possible shortcut nonetheless though, as a safety measure. It'd evaluate to `None` which is falsy, and even though it's unlikely (currently not used in any of our projects) I'd rather keep it

Reply:
In theory we should also cover the case where all options `.is_hidden` is `"true"` but let's not as that really is no use case.